packages/kmod-6.12-amdgpu/kmod-6.12-amdgpu.spec

```
@@ -0,0 +1,81 @@
+%global kernel_sources %{_cross_usrsrc}/kernels/6.12
+%define _kernel_version %(cat %{kernel_sources}/include/config/kernel.release)
+%global _cross_kmoddir %{_cross_libdir}/modules/%{_kernel_version}
+%global _ko ko
+
+# Cross-compilation make macro for kernel builds
+%global kmake %{shrink: \
+make -s \
+  ARCH="%{_cross_karch}" \
+  CROSS_COMPILE="%{_cross_target}-" \
+  INSTALL_HDR_PATH="%{buildroot}%{_cross_prefix}" \
+  INSTALL_MOD_PATH="%{buildroot}%{_cross_prefix}" \
+  INSTALL_MOD_STRIP=1 \
+  %{nil}}
+
+Name: %{_cross_os}kmod-6.12-amdgpu
+Version: 30.10
+Release: 1%{?dist}
+Summary: AMD GPU drivers for the 6.12 kernel
+License: GPL-2.0 WITH Linux-syscall-note
```

```suggestion
License: MIT AND GPL-2.0-only AND (GPL-2.0-only WITH Linux-syscall-note) AND GPL-2.0-or-later AND (GPL-2.0-or-later WITH Linux-syscall-note) AND LGPL-2.0-or-later
```

Derived from this list:
```
❯ rg -N -I 'SPDX-License-Identifier'|sed 's,[/\*#],,g'|awk '{$1=$1};1'|sort -u
SPDX-License-Identifier: BSD-3-Clause OR GPL-2.0
SPDX-License-Identifier: GPL-2.0
SPDX-License-Identifier: GPL-2.0+
SPDX-License-Identifier: GPL-2.0 AND MIT
SPDX-License-Identifier: GPL-2.0-only
SPDX-License-Identifier: GPL-2.0-or-later
SPDX-License-Identifier: GPL-2.0 OR MIT
SPDX-License-Identifier: GPL-2.0+ WITH Linux-syscall-note
SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-3-Clause)
SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) OR MIT
SPDX-License-Identifier: LGPL-2.0+
SPDX-License-Identifier: MIT
```

`BSD-3-Clause` omitted because we might as well pick `GPL-2.0-only` for that case.

---

Will update this